src/pybind/mgr/orchestrator/module.py

```diff
@@ -1190,16 +1190,19 @@ def _apply_rgw(self,
 
     @_cli_write_command(
         'orch apply ha_rgw',
```

Comment:
```suggestion
        'orch apply ha-rgw',
```

let's not introduce underscores here.